web-client/src/modules/phone/pages/ContentPage.tsx

```diff
@@ -12,7 +12,7 @@ import { PhoneVerifyLocation } from './routes/PhoneVerifyRoute/constants';
 import PhoneVerifyRoute from './routes/PhoneVerifyRoute/PhoneVerifyRoute';
 
 const ContentPage = (): ReactElement => {
-  const user: firebase.User = useSelector((state: AppState) => state.auth.user);
+  const user = useSelector((state: AppState) => state.auth.user);
```

Comment:
shouldn't we enforce type here? it would help us while accessing the properties right?

Reply:
No, I refactored the store implementation and, right now, you don't need to cast the type of state properties. They are already recognized of being of that type.

Reply:
okay sounds good